IPython/html/static/notebook/js/quickhelp.js

```diff
@@ -203,6 +207,34 @@ define([
         this.events.on('rebuild.QuickHelp', function() { that.force_rebuild = true;});
     };
 
+    QuickHelp.prototype.build_key_names = function () {
+       var key_names_mac =  [{ shortcut:"⌘", help:"Command" },
+                    { shortcut:"⌃", help:"Control" },
+                    { shortcut:"⌥", help:"Option" },
+                    { shortcut:"⇧", help:"Shift" },
+                    { shortcut:"↩", help:"Return" },
+                    { shortcut:"␣", help:"Space" },
+                    { shortcut:"⇥", help:"Tab forward" },
+                    { shortcut:"⇤", help:"Tab back" }];
```

Comment:
I don't think "Tab back" is a key. It's certainly not a key we use, so it should probably be removed and "Tab forward" should be "Tab".